argo/base_v3/kustomization.yaml

```diff
@@ -17,10 +17,10 @@ commonLabels:
 images:
 - name: argoproj/argoui
   newName: argoproj/argoui
-  newTag: v2.3.0
+  newTag: v2.4.3
 - name: argoproj/workflow-controller
   newName: argoproj/workflow-controller
-  newTag: v2.3.0
+  newTag: v2.7.5
```

Comment:
Is this change necessary for Azure?
I'm a bit reluctant to make this change in this PR.
How is this going to affect Kubeflow Pipelines users? Is Pipelines compatible with this Argo version?
I think we should split this into a separate PR and validate compatibility.

Reply:
Ok. I took it out. I will create a separate PR for that